pkg/util/netops/netops.go

```diff
@@ -54,17 +67,17 @@ func NewNSFromPath(nsPath string) (*NS, error) {
 	if err != nil {
 		return nil, fmt.Errorf("failed to get a handle for network namespace %s: %w", nsPath, err)
 	}
-	ns := &NS{
-		Name:     name,
-		Path:     nsPath,
+	ns := &namespace{
+		name:     name,
+		path:     nsPath,
 		nsHandle: nsHandle,
 		handle:   handle,
 	}
 	return ns, nil
 }
 
-// GetNS returns an NS of the current network namespace
-func GetNS() (*NS, error) {
+// CurrentNamespace returns the current network namespace
```

Comment:
Is GetCurrentNamespace a better alternative ?

Reply:
Thank you for your comments.

I renamed this function to `OpenCurrentNamespace`, because I want to emphasize that a Namespace must be closed.